server/protocols/protocols-library/src/test/resources/testServerPlain.xml

```diff
@@ -0,0 +1,9 @@
+<testerver enabled="true">
+    <jmxName>testserver-custom</jmxName>
+    <bind>0.0.0.0:25</bind>
```

Comment:
```suggestion
    <bind>0.0.0.0:0</bind>
```

Bonding a non fixed port is cleaner as it ensures the build can run several time concurrently on the same computer.

Port can then be retrieved programmatically in the tests.

Reply:
OK. This is supposed to be a configuration test only, so the address is never used anyway. I would have liked to have something like 1.2.3.4:5 here, but since the bind address is unfortunately not testable, I might as well keep it all zeroes.